src/components/form/Contact.js

```diff
@@ -54,34 +54,60 @@ const ContactForm = props => {
           </H3>
           <P>
             The best way to contact us is by emailing us at{' '}
-            <Link to="mailto:hello@reactgrahql.academy">
+            <Link
+              to="mailto:hello@reactgrahql.academy"
```

Comment:
Can you add some love here?
- `?subject=`
- `&body=`

⚠️ You need to encode as HTML entities
example:
https://github.com/flexbox/davidl/blob/9a3a3cc2a8d5202f1d0e514dfca40c68b9f2c997/source/layouts/_quick-contact.html.slim#L7

Reply:
:+1:

Reply:
:+1: